src/api/kube.ts

```diff
@@ -1150,13 +1150,18 @@ export class KubeHelper {
         yamlCr.spec.auth.updateAdminPassword = true
       }
       if (flags.tls) {
+        if (!yamlCr.spec.k8s) {
+          yamlCr.spec.k8s = {}
+        }
         yamlCr.spec.server.tlsSupport = flags.tls
         if (!yamlCr.spec.k8s.tlsSecretName) {
           yamlCr.spec.k8s.tlsSecretName = 'che-tls'
         }
       }
       yamlCr.spec.server.selfSignedCert = flags['self-signed-cert']
-      yamlCr.spec.k8s.ingressDomain = flags.domain
+      if (flags.domain) {
```

Comment:
```suggestion
        if (!yamlCr.spec.k8s) {
          yamlCr.spec.k8s = {}
        }
      if (flags.domain) {
```
Sorry, I forgot one more changes.

Reply:
```suggestion
```